src/lwc/bdiObjectMappingModal/bdiObjectMappingModal.js

```diff
@@ -202,7 +197,7 @@ export default class bdiObjectMappingModal extends LightningElement {
     /*******************************************************************************
     * @description Handles escape key press and closes the modal
     */
-    escapeFunction(event) {
+    escapeFunction = (event) => {
```

Comment:
@psadie-sfdo I've included a little fix for an error I was getting when accessing Advanced Mapping while debug mode was turned on. The error was occurring when we tried to re-assign escapeFunction in the constructor, and the message said that `this.escapeFunction` is read-only.

By changing `escapeFunction` to an arrow function, we no longer have to re-bind that function because arrow functions are automatically bound to the current scope.

Reply:
Thanks!! I believe this would then fix an issue raised by Michael Beaty here #4912. If so, could we go ahead and also include that issue in the 'Issues Closed' section?